modules/FhirAPI/src/FhirAPI/FhirRestApiBuilder/Parts/Search/SearchStrategies/ServiceRequestSearch.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace FhirAPI\FhirRestApiBuilder\Parts\Search\SearchStrategies;
+
+
+class ServiceRequestSearch extends BaseSearch
+{
+    public $paramsToDB = array();
+    public $MAIN_TABLE = 'fhir_service_request';
+    public function search()
+    {
+
+        if(!empty($this->searchParams)){
+            foreach($this->searchParams as $index => $data){
+                if($index === "occurrence" || $index === "authored"){
+                    foreach($data as $pos => $record){
+                        $this->searchParams[$index][$pos]['value']=$this->fhirObj->convertToDateTime($record['value']);
```

Comment:
add a comment regarding the format date we support currently

Reply:
sure